packages/lesswrong/components/common/excerpts/CommentExcerpt.tsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import { Components, registerComponent } from "../../../lib/vulcan-lib";
 import { commentGetPageUrlFromIds } from "../../../lib/collections/comments/helpers";
+import type { CommonExcerptProps } from "./ContentExcerpt";
 
-const CommentExcerpt = ({comment, lines = 3, className}: {
+const CommentExcerpt = ({
+  comment,
+  ...commonExcerptProps
+}: CommonExcerptProps & {
```

Comment:
The changes here and in `TagExcerpt` just allow us to automatically accept the various styling props in all the various excerpt types.